### ISSUE TITLE
Textile care symbols [non-emoji!]

### ISSUE THREAD

**Issue by spixi** [Crissov/unicode-proposals#410]
Proposal for new characters
-------------------
| Name | Description |
|---|---|
| TEXTILE CARE SYMBOL WASHING TUB | washing, shows a tub |
| TEXTILE CARE SYMBOL WASHING TUB WITH THIRTY DEGREES | |
| TEXTILE CARE SYMBOL WASHING TUB WITH FOURTY DEGREES | |
| TEXTILE CARE SYMBOL WASHING TUB WITH FIFTY DEGREES | |
| TEXTILE CARE SYMBOL WASHING TUB WITH SIXTY DEGREES | |
| TEXTILE CARE SYMBOL WASHING TUB WITH SEVENTY DEGREES | |
| TEXTILE CARE SYMBOL WASHING TUB WITH NINETY-FIVE DEGREES | |
| TEXTILE CARE SYMBOL WASHING WITH HAND | hand washing |
| TEXTILE CARE SYMBOL SQUARE | drying |
| TEXTILE CARE SYMBOL SQUARE WITH BOW IN THE TOP | clothes line |
| TEXTILE CARE SYMBOL SQUARE WITH HORIZONTAL STROKE | flat dry |
| TEXTILE CARE SYMBOL SQUARE WITH TRIPLE VERTICAL STROKE | drip dry |
| TEXTILE CARE SYMBOL SQUARED CIRCLE | machine drying, shows a drier |
| TEXTILE CARE SYMBOL IRON | shows an iron |
| TEXTILE CARE SYMBOL CIRCLE | chemical cleaning |
| TEXTILE CARE SYMBOL CIRCLED LATIN CAPITAL LETTER A | all cleaning agents  |
| TEXTILE CARE SYMBOL CIRCLED LATIN CAPITAL LETTER F | hydrocarbon solvent |
| TEXTILE CARE SYMBOL CIRCLED LATIN CAPITAL LETTER P | tetrachloroethene |
| TEXTILE CARE SYMBOL CIRCLED LATIN CAPITAL LETTER W | wet cleaning |
| TEXTILE CARE SYMBOL TRIANGLE | bleaching |
| TEXTILE CARE SYMBOL TRIANGLE WITH CL | bleaching with chlorine |
| TEXTILE CARE SYMBOL TRIANGLE WITH DOUBLE SOLIDUS | bleaching without chlorine |
| TEXTILE CARE SYMBOL COMBINING HEAVY DOT | low temperature |
| TEXTILE CARE SYMBOL COMBINING DOUBLE HEAVY DOT | medium temperature |
| TEXTILE CARE SYMBOL COMBINING TRIPLE HEAVY DOT | high temperature |
| TEXTILE CARE SYMBOL COMBINING QUADRUPLE HEAVY DOT | very high temperature |
| TEXTILE CARE SYMBOL COMBINING CROSS | do not (wash/iron/bleach/tumble) |
| TEXTILE CARE SYMBOL COMBINING LINE BELOW | gentle |
| TEXTILE CARE SYMBOL COMBINING DOUBLE LINE BELOW | very gentle |
| TEXTILE CARE SYMBOL COMBINING DOUBLE SOLIDUS IN THE LEFT TOP CORNER | dry in the shadow |

Examples
-------------------
![b-bedruckte_baumwolletiketten-etiketten_aus_baumwolle-naturbaumwolle-oekobaumwolle-12](https://user-images.githubusercontent.com/3192596/37242786-5c36d218-246f-11e8-8ef6-63b42dc90410.jpg)
![pflegesymbole-trockner](https://user-images.githubusercontent.com/3192596/37242788-5c71e1aa-246f-11e8-916d-f72cbe960862.jpg)
![pflegesymbole-bleichen](https://user-images.githubusercontent.com/3192596/37242789-5c8f8eb2-246f-11e8-9abb-d5e20397135d.jpg)
![pflegesymbole-reinigung](https://user-images.githubusercontent.com/3192596/37242790-5cda8a20-246f-11e8-9d1e-84180392a57c.jpg)
![pflegesymbole-buegeln-g](https://user-images.githubusercontent.com/3192596/37242791-5cf5f102-246f-11e8-835d-72add53ee288.jpg)
![erklaerung-pflegesymbole](https://user-images.githubusercontent.com/3192596/37242792-5d142ae6-246f-11e8-98d5-3bdfb6544bd9.jpg)
![mb0214_waschsymbole 7](https://user-images.githubusercontent.com/3192596/37242787-5c53924a-246f-11e8-9098-d7db3f6c361b.jpg)
![drying](https://user-images.githubusercontent.com/3192596/37243524-0b2a0104-247b-11e8-88b6-32ead85a20f7.png)

Wash Care Symbols M54
-------------------
The font Wash Care Symbols M54 has already implemented textile care symbols. However, they are encoded in the _C0 Controls and Basic Latin_ block.
![mp1_wash-care-symbols-m54_1](https://user-images.githubusercontent.com/3192596/37242785-5c0242e6-246f-11e8-909d-f2db04b4d818.png)

Standard References
-------------------
- [How to propose emojis to Unicode](http://unicode.org/emoji/selection.html)
- [How to propose other additions or changes to Unicode](http://www.unicode.org/pending/proposals.html) ([required form](http://www.unicode.org/L2/summary.html), email address: [docsubmit＠unicode.org](mailto:docsubmit@unicode.org?subject=UTC%20Document%20Submission:%20{name%20of%20your%20topic}&body=Please%20consider%20the%20attached%20document%20for%20submission%20to%20the%20UTC))
- [Unicode document submission form](http://www.unicode.org/reporting.html)

**Comment by Crissov** [comment]
I think I remember having seen a failed proposal to encode some of them , will need to check.

**Comment by spixi** [comment]
@Crissov Do you mean that mail here?
http://unicode.org/mail-arch/unicode-ml/y2003-m06/0274.html

**Comment by Crissov** [comment]
This could very well be it. I think I was subscribed to the mailing list back then.

**Comment by nbeaver** [comment]
Note also [ISO 3758](https://www.iso.org/standard/42918.html), which [dates back to 1991](https://www.iso.org/standard/9256.html).

**Comment by PabloReszczynski** [comment]
Forgive me if this has been asked before, but aren't these symbols trademarked? Would font makers, or Unicode Inc. require a license to reproduce these symbols?

> The five care symbols are protected trademarks in most countries. They may not be reproduced, issued or used without a special license agreement with GINETEX, co-owner of the trademarks together with COFREET. This Intellectual Property license agreement commits companies to use the symbols correctly at all times.

[Source](https://www.ginetex.net/GB/labelling/labelling.asp)

**Comment by 8HoLoN** [comment]
There is a lack of the unit in the name column : THIRTY DEGREES => THIRTY DEGREES CELSIUS

**Comment by spixi** [comment]
@PabloReszczynski Good point, but I am pretty sure that the glyphs do not have enough threshold of originality for a copyright. Regarding trademark right: GINETEX only has a trademark on the combination of the five care symbols in one picture and this also only applies for certain Nizza classes (e. g. leaflets about textile care, marketing and consulting services, but not for the textiles themselves).

**Note that the protectability of the textile care symbols may vary between different countries!**

For example, for Germany I found:
https://register.dpma.de/DPMAregister/marke/register/3020090190313/DE (Nizza classes 35, 37, 41, 42, 45)
https://register.dpma.de/DPMAregister/marke/register/3020180219248/DE (Nizza classes 9, 37, 40, 41, 45)
https://register.dpma.de/DPMAregister/marke/registerIR?AKZ=849319 (covers all kinds of textile products, non-textile labels, information and consulting services and software, not protectable in Germany, **but possibly in other European countries**)
https://register.dpma.de/DPMAregister/marke/registerIR?AKZ=849320 (covers all kinds of textile products, non-textile labels, information and consulting services and software, not protectable in Germany, **but possibly in other European countries**)

@8HoLoN The glyphs are also lacking a unit. Character descriptions should match the glyph appearance and not add additional semantics.

**Comment by 8HoLoN** [comment]
@spixi I strongly disagree, unit is crucial, the fact that the glyph does not contain the `C` is not relevant because some of glyph does not include `°` character meanwhile the name includes `Degrees`. So, to get full meaning you should get the unit.

**Comment by spixi** [comment]
> 
> 
> @spixi I strongly disagree, unit is crucial, the fact that the glyph does not contain the `C` is not relevant because some of glyph does not include `°` character meanwhile the name includes `Degrees`. So, to get full meaning you should get the unit.

@8HoLoN I understand, that's a good argument. I found another reference sheet which also uses the dotted annotation used in the USA and Canada. It also has four additional symbols for dry cleaning, I have never seen before. They are also included in Wash Care Symbols M54. Does someone know if they are also part of ISO 3758 or any national standard?
![Wash_Care_symbols](https://user-images.githubusercontent.com/3192596/115743875-59147e00-a392-11eb-8978-ad20aeefa4fe.jpg)

**Comment by spixi** [comment]
I added the Celsius and also the four tangents for additional dry-cleaning instructions.

**Comment by idontusenumbers** [comment]
A lot of these seem like a good match for ligatures or modifiers; having a code point per temperature seems unsustainable/unnecessary.

**Comment by spixi** [comment]
> 
> 
> A lot of these seem like a good match for ligatures or modifiers; having a code point per temperature seems unsustainable/unnecessary.

The temperature values are fixed and have different representations (some countries use numeric values, others use dots).

**Comment by spixi** [comment]
Most of the proposed characters are also included in ISO 7000 / IEC 60417 Graphical symbols (#443)
https://www.iso.org/obp/ui#iso:pub:PUB400008:en
Symbol numbers 3085 to 3125.
